### PR TITLE
Run integration tests for PRs

### PR DESCRIPTION
**Pull request by HaraldNordgren**
Unbreak the integration tests syntax.

Follow-up to https://github.com/sashabaranov/go-openai/pull/820.

### CODE REVIEW

**Comment by HaraldNordgren** [comment]
Ping @sashabaranov @tylergannon @eiixy

**Comment by eiixy** [comment]
> Ping @sashabaranov @tylergannon @eiixy

I've already added it here. #819 
https://github.com/sashabaranov/go-openai/blob/290bc29284bf65e879d76fee57f15c7d4aaaebef/api_integration_test.go#L223

**Comment by sashabaranov** [comment]
Thank you for the PR!

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+  pull_request:
```
Comment (sashabaranov) [inline .github/workflows/integration-tests.yml:7]: There's a tricky balance here — allowing integration tests in PRs will add a risk to spending the OpenAI budget by everyone, basically. Let's remove this for now

**Review by sashabaranov — APPROVED** [review]

**Pull request merged**